raiutils/raiutils/data_processing/data_processing_utils.py

```diff
@@ -143,7 +143,7 @@ def serialize_json_safe(o: Any):
         if isinstance(o, str):
             return json.dumps(o)[1:-1]
         return o
-    elif isinstance(o, datetime.datetime):
+    elif isinstance(o, datetime.datetime) or isinstance(o, pd.Timestamp):
```

Comment:
Could you have a separate PR for this in raiutils? Also add a test case for this is raiutils.

Reply:
#2320